src/obb_demo/processor.cljs

```diff
@@ -62,7 +62,9 @@
     #_(println "game" (simplifier/clean-result {:board game}))
     (println "--" player actions)
     (if (= :final (game/state game))
-      {:game (deployed-game {:mode (get game-data :mode :annihilation)})}
+      (let [mode (get game-data :mode :annihilation)]
```

Comment:
Another place where we set the game mode explicitly (I assume that we want the default value here).